shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/rulealtered/RuleAlteredJobScheduler.java

```diff
@@ -17,17 +17,18 @@
 
 package org.apache.shardingsphere.data.pipeline.scenario.rulealtered;
 
+import java.util.Collection;
+import java.util.LinkedList;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
```

Comment:
You could import code style schema `src/resources/code-style-idea.xml` into IDEA to keep standard importing.